### PR TITLE
Fix: Truncate dagster dataframe previews

### PR DESCRIPTION
**Pull request by bnchrch**
## Overview
Fixes an issue where the dataframe preview was too large
image

### CODE REVIEW

**Review by erohmensing — APPROVED** [review]
Noice :shipit:

File: airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_report.py [hunk 1]
```diff
@@ -324,7 +324,6 @@ def connector_registry_report(context, all_destinations_dataframe, all_sources_d
 
     metadata = {
         "first_10_preview": MetadataValue.md(all_connectors_dataframe.head(10).to_markdown()),
-        "json": MetadataValue.json(json_string),
```
Comment (erohmensing) [inline airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_report.py:327]: what's going missing here?

Comment (bnchrch) [reply airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_report.py:327]: Exactly, we had the raw json there essentially for fun, but it seems thats a risk to the asset completing successfully. Now you just have to click the url below instead to see the same data

Comment (erohmensing) [reply airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_report.py:327]: word, that works!

**Pull request merged**